### DISCUSSION TITLE
Android 版 Yandex 浏览器使用 Simpread 经验

### DISCUSSION THREAD

**Discussion by mmqmzk** [Kenshin/simpread#3015] [使用技巧] [answered]
# 前提

Android 版的 Yandex 浏览器支持部分 Chrome 扩展, 但是**直接到商店安装 Simpread 会提示不兼容, 无法安装**。

经本人测试, 是**由于 Android 版 Yandex 不支持 Chrome 扩展的 TTS (Text to Speech/文本转语音) API 导致**的。

# 安装方法

下载 Simpread 扩展的 crx  文件, 解压后放到 Android 设备内部存储的目录: `Android/data/com.yandex.browser/` 下面的任意子目录, 修改 `manifest.json` 文件, 去掉 `permissions` 中的 `tts` 权限。

打开 Yandex, 访问 `chrome://extensions`, 打开页面右上的开发者模式, 点击 `加载已解压的扩展程序`, 导航到刚才修改的 `manifest.json`  文件选择, 即可加载 Simpread。

# 使用体验

使用体验跟 Kiwi 浏览器差不多, TTS 功能当然是用不了的。

# 注意

有一些系统自带的文件管理器禁止访问 `Android/data`, 可以用第三方文件管理器。

注: Simpread 的标注等高级功能是无法使用的, 不要激活高级账号。

# 引申

在 Kiwi Browser 安装和使用简悦的说明和教程，[请看这里](https://zhuanlan.zhihu.com/p/415622477)。

**Comment by Kenshin** [comment, marked as answer]
嗨，谢谢提供 Yandex 的使用体验。🙏

关于简悦在 Android 使用的体验，可以看这篇文章 https://zhuanlan.zhihu.com/p/415622477

***

> 标注等高级功能的问题

在 Kiwi 上是可以使用的，因此在 Yandex 应该也是一样的，细节上面链接中给出了，我把图片放到了下面

![image](https://user-images.githubusercontent.com/81074/140595616-a633d54f-b50c-4fd2-8bd1-dff5b0a0a2e1.png)

所以，你可以试试。如果确实无法使用的话，可以描述下无法使用的原因。我会补充到上面这个链接中。

***

PS：我把你的内容简单的修改了下（标点和加段落）。
